fem/ceed/interface/ceed.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_LIBCEED_CEED
+#define MFEM_LIBCEED_CEED
+
+#ifdef MFEM_USE_CEED
+#include <ceed.h>
+#if !CEED_VERSION_GE(0,10,0)
+#error MFEM requires a libCEED version >= 0.10.0
+#endif
+#endif
+
+namespace mfem
+{
+
+namespace internal
+{
+
+#ifdef MFEM_USE_CEED
+extern Ceed ceed;
+#endif
+
+} // namespace internal
+
+namespace ceed
+{
+
+} // namespace ceed
```

Comment:
Is this empty namespace doing anything?

Reply:
Removed in [8197122](https://github.com/mfem/mfem/pull/2952/commits/819712212a7cec2986110a3c47e0042cc5696ae5).